Angular/login/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AddExchangeComponent } from './components/add-exchange/add-exchange.component';
+import { CompanyComponent } from './components/company/company.component';
+import { DashboardComponent } from './components/dashboard/dashboard.component';
+import { ExchangeComponent } from './components/exchange/exchange.component';
+import { HomeComponent } from './components/home/home.component';
+import { IpoComponent } from './components/ipo/ipo.component';
+import { LoginComponent } from './components/login/login.component';
+import { NavbarComponent } from './components/navbar/navbar.component';
+import { SectorComponent } from './components/sector/sector.component';
+import { SignupComponent } from './components/signup/signup.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405403)
```suggestion
import { SignupComponent } from "./components/signup/signup.component";
```